### ISSUE TITLE
【要望】API バージョン情報を確認できるエンドポイントの追加

### ISSUE THREAD

**Issue by supermomonga** [kabucom/kabusapi#605]
## 要望の内容

API 仕様のバージョンを取得するための API エンドポイントを追加して頂きたいです。

イメージとしましては、`http://localhost:18080/kabusapi/api_version` に GET リクエストを送信すると、以下の様なレスポンスが取得できる、といった様なものとなります。

```json
{
    "Version": "1.5"
}
```

- ここで返却されるバージョン値は、[OpenAPI 仕様書の `version` 項目](https://github.com/kabucom/kabusapi/blob/516fc356f74f9f5470f47957ffdceda8041221cf/reference/kabu_STATION_API.yaml#L3)を想定しております
- API の仕様が変更されるたび、OpenAPI 仕様書の `version` 項目の値が変化することを前提としております
- この API エンドポイントは認証不要（`X-API-KEY` リクエストヘッダ不要）で利用できることを想定しております

## 背景

バージョンアップにより API の仕様が変更された際、プログラムから検知を行いたい。プログラム作成時とは API の仕様が異なる場合、意図しない挙動となる可能性があるため。

## 利用例

自作の自動売買プログラムにおいて、起動時にまず `http://localhost:18080/kabusapi/api_version`  エンドポイントを利用し API バージョンを返却値として得る。

返却値がプログラムが想定している API バージョンと異なる場合、プログラム作成時とは API の仕様が異なることを意味するため、意図しない挙動を防ぐためにプログラムを終了させる。

返却値がプログラムが想定している API バージョンと同一の場合、プログラム作成時から API の仕様が変化していないとみなし、自動売買プログラムの実行を継続する。

**Comment by yasuyuki-nakazawa** [comment]
APIの変更をプログラム起点で認識したいというニーズですね。
システム側にも連携します。

**Comment by supermomonga** [comment]
> APIの変更をプログラム起点で認識したいというニーズですね。

はい、そうなります。単に変更の有無のみでなく API バージョン番号を得たい理由は、git リポジトリにおける `kabu_STATION_API.yaml` の変更履歴と照らし合わせて変更内容を確認するためです。
